cfme/tests/junit-report.xml

```diff
@@ -0,0 +1 @@
+<?xml version="1.0" encoding="utf-8"?><testsuite errors="0" failures="0" name="pytest" skips="0" tests="0" time="2105.573"></testsuite>
```

Comment:
**REQUIRED:** Why is this in here?

Reply:
I thought you might be interested in my test results :).
sorry, it was accidentally added.